vm-setup.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# This script is a customization Script and it is used to set instances launched using openstack. 
+
+# update things.
+sudo su
+apt-get -y update && apt-get upgrade -y
+
+# install maven, pidstat and htop. 
+apt install maven -y
+apt install sysstat -y
+apt install htop -y
+
+# install wrk2.
+apt-get install build-essential libssl-dev git -y
+git clone https://github.com/giltene/wrk2.git
+cd wrk2
+make
+cp wrk /usr/local/bin
+cd ../
+
+# install openjdk.
+apt-get install openjdk-8-jdk -y
```

Comment:
Why installing 8 and 9? And could you please make sure the scripts checks which is being used?

Reply:
I had a new idea thanks to you, thanks!! I can install only openjdk9 and be happy. We will use only one, right? I have notice until now, after install a version of a jdk, the jdk being used is the last one installed. We can see it using a "java -version" or "update-alternatives --config java". The last one gives me the option to choose which jdk to use.